gdpr/anonymizers/fields.py

```diff
@@ -0,0 +1,125 @@
+import warnings
+from datetime import timedelta
+from typing import Any, Callable
+
+from django.core.exceptions import ImproperlyConfigured
+
+from gdpr.anonymizers.base import FieldAnonymizer
+
+
+class FunctionAnonymizer(FieldAnonymizer):
+    """
+    Use this field anonymization for defining in situ anonymization method.
```

Comment:
in situ?

Reply:
To je latinsky znamená to v místě. Vycházel jsem z toho, že jsou třídící algoritmy in situ(in place) což je jakože to používá původní datovou strukturu....

Opraveno